pom.xml

```diff
@@ -29,13 +29,25 @@
       <!-- I used to work for MITRE for many years but I don't anymore. -->
       <!--<organization>MITRE</organization>-->
     </developer>
+    <developer>
+      <name>Prasad Madhbhavikar</name>
+      <email>prasad.madhbhavikar@gmail.com</email>
+      <!-- Currently working with Cisco and the changes are as requested -->
+      <organization>Cisco</organization>
+    </developer>
+    <developer>
+      <name>Gourav Jangra</name>
+      <email>jangra.gourav143@gmail.com</email>
+      <!-- Currently working with Cisco and the changes are as requested -->
+      <organization>Cisco</organization>
+    </developer>
   </developers>
 
   <scm>
     <url>https://github.com/dsmiley/HTTP-Proxy-Servlet</url>
     <connection>scm:git:https://dsmiley@github.com/dsmiley/HTTP-Proxy-Servlet.git</connection>
     <developerConnection>scm:git:git@github.com:dsmiley/HTTP-Proxy-Servlet.git</developerConnection>
-    <tag>HEAD</tag>
+    <tag>smiley-http-proxy-servlet-1.9</tag>
```

Comment:
hmm; I'm not sure what the implication of this change is... but I see it's something to maintain between versions :-/